cdap-e2e-tests/src/e2e-test/features/tethering/TetheringRegistration.feature

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Tethering_Registration
+Feature: Tethering registration and management
+
+  @TETHERING_CREATION_TEST
+  Scenario: Validate successful creation of new tethering connection requests
+    Given Open tethering client Datafusion project
+    When Navigate to tethering page
+    Then Open create new request page
+    Then Click to select a namespace
+    Then Enter project name "test-project"
+    Then Enter region "us-west-1b"
+    Then Enter instance name "test"
+    Then Enter instance url for tethering server
+    Then Enter description "test description"
+    Then Finish creating new tethering request
+    Then Verify the request was created successfully
+
+  @TETHERING_CREATION_TEST
+  Scenario: Validate unsuccessful creation of a new request with the same instance name
+    Given Open tethering client Datafusion project
+    When Navigate to tethering page
+    Then Open create new request page
+    Then Click to select a namespace
+    Then Enter project name "test-project"
+    Then Enter region "us-west-1b"
+    Then Enter instance name "test"
+    Then Enter instance url for tethering server
+    Then Enter description "test description"
+    Then Finish creating new tethering request
+    Then Verify the request failed to be created
```

Comment:
Looks like scenarios are not independent on each other (this one depends on the previous scenario). Seems like that could result in failures if we specify some tags (e.g: TETHERING_CONNECTION_MANAGEMENT) in the TestRunner.

Reply:
added TETHERING_CONNECTION_MANAGEMENT tag to the first scenario so all the tests can run if only that tag is specified